src/styles/home.scss

```diff
@@ -133,8 +133,56 @@
   }
 }
 
+.home-header-story {
+  @include media("<=phone") {
+    display: block;
+  }
+
+  @include media(">=phone", '<tablet') {
+    display: none;
+  }
+
+  @include media('>=tablet', '<smalldesktop') {
+    display: none;
+  }
+
+  @include media('>=smalldesktop', '<desktop') {
+    display: none;
+  }
+
+  @include media(">=desktop", '<largedesktop') {
+    display: none;
+  }
+
+  @include media('>=largedesktop') {
+    display: none;
+  }
+}
+
 .home-3d-model {
-  min-height: 250px;
+  @include media("<=phone") {
+    display: none;
+  }
+
+  @include media(">=phone", '<tablet') {
```

Comment:
@include media(">=phone", '>=largedesktop')

i think instead of caparing all breakpoints u can do something like this